SPECS/kernel/config

```diff
@@ -330,46 +339,34 @@ CONFIG_CC_HAS_SANE_STACKPROTECTOR=y
 CONFIG_SMP=y
 CONFIG_X86_FEATURE_NAMES=y
 CONFIG_X86_X2APIC=y
-# CONFIG_X86_MPPARSE is not set
+CONFIG_X86_MPPARSE=y
 # CONFIG_GOLDFISH is not set
-# CONFIG_X86_CPU_RESCTRL is not set
+CONFIG_X86_CPU_RESCTRL=y
 CONFIG_X86_EXTENDED_PLATFORM=y
-# CONFIG_X86_NUMACHIP is not set
+CONFIG_X86_NUMACHIP=y
 # CONFIG_X86_VSMP is not set
-# CONFIG_X86_UV is not set
+CONFIG_X86_UV=y
 # CONFIG_X86_GOLDFISH is not set
-CONFIG_X86_INTEL_MID=y
+# CONFIG_X86_INTEL_MID is not set
 CONFIG_X86_INTEL_LPSS=y
-# CONFIG_X86_AMD_PLATFORM_DEVICE is not set
+CONFIG_X86_AMD_PLATFORM_DEVICE=y
 CONFIG_IOSF_MBI=y
-# CONFIG_IOSF_MBI_DEBUG is not set
+CONFIG_IOSF_MBI_DEBUG=y
```

Comment:
used for debug, keep off for now

https://cateee.net/lkddb/web-lkddb/IOSF_MBI_DEBUG.html